src/images/icons/add.svg

```diff
@@ -1 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" aria-hidden="true" focusable="false" class="Icon Icon--plus"><g fill-rule="evenodd"><path fill="none" fill-rule="evenodd" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 8V3v5H3h5zm0 0v5-5h5-5z"></path><rect fill="none" stroke="none" x="0" y="0" width="16" height="16"></rect></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Note that both `aria-hidden` and `focusable=false` are removed for all of our SVGs in this PR. You can read up about how `aria-hidden` can be used for enhanced a11y here: https://css-tricks.com/accessible-svgs/ but the upshot from my perspective is that we _don't_ want it here. The reason is that our icons are used standalone in many/most cases (i.e. without supporting visible text), and thus are "meaningful" icons, and shouldn't thus be hidden from screen readers, IMHO. This is worthy of discussion! Please provide feedback if you have any.

I've also gone ahead and removed `class` attrs.

Reply:
Were these modifications done manually? If so I see a future development hazard here that developers may not realize they need to make these changes when adding new icons or update an existing icon using an export from eg. Sketch. One way to resolve this would be to deal with this dynamically at runtime. Another would be to semi-automate the changes somehow.